app/views/admin/organizations/custom_layouts/_public_image.html.haml

```diff
@@ -0,0 +1,18 @@
+.card.bg-light.admin-public-image
```

Comment:
Is there a reason this needs to be a different template than the blogs template?

If the only reason is that this shouldn't include the delete button, can you just hide the delete button via css?

Reply:
I didn't really want to link this to public_image template in news, since it felt sort of weird. With `public` it was a repo where a lot of things looked, so that made sense. What if I made a public_image_revised and added it to the public folder? Would that work? It would be really easy to make it the same template, I just want to put it somewhere that makes sense.

Reply:
What about `_admin_public_image`? I think both the blog and this should reference the same partial.

Reply:
Yeah, I was thinking about it and this was pretty much what I came to, also. This is what I'll go with!